rtos/rtx/TARGET_CORTEX_A/RTX_CM_lib.h

```diff
@@ -50,7 +50,7 @@
 #define _declare_box(pool,size,cnt)  uint32_t pool[(((size)+3)/4)*(cnt) + 3]
 #define _declare_box8(pool,size,cnt) uint64_t pool[(((size)+7)/8)*(cnt) + 2]
 
-#define OS_TCB_SIZE     52
+#define OS_TCB_SIZE     60
```

Comment:
Hi.

Code of review test is completed.There is no problem.

Regards,
Yamanaka